src/jit/emit.h

```diff
@@ -943,7 +943,7 @@ class   emitter
             }
 
             // Note that we use the _idReg3 and _idReg4 fields to hold
-            // the live gcrefReg mask for the call instructions on arm
+            // the live gcrefReg mask for the call instructions on arm64
```

Comment:
(Minor) The comment in emitNewInstrCallInd() and emitNewInstrCallDir in emit.cpp needs fixing.  It reads as

/* Save the the live GC registers in the unused 'idReg/idRg2' fields */